### PR TITLE
Real Time Data forward- Update ODE to accept new BSM data format from V2xHub

### PR DESCRIPTION
**Pull request by dan-du-car**
https://github.com/usdot-jpo-ode/jpo-ode/issues/398

### CODE REVIEW

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java [hunk 1]
```diff
@@ -127,6 +127,7 @@
    private String kafkaTopicOdeBsmTxPojo = "topic.OdeBsmTxPojo";
    private String kafkaTopicOdeBsmDuringEventPojo = "topic.OdeBsmDuringEventPojo";
    private String kafkaTopicFilteredOdeBsmJson = "topic.FilteredOdeBsmJson";
+	private String kafkaTopicOdeRawEncodedMessageJson = "topic.OdeRawEncodedMessageJson";
```
Comment (TonyEnglish) [inline jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java:130]: Fix spacing on line begin to match file standard

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java [hunk 2]
```diff
@@ -626,5 +627,14 @@ public String getKafkaTopicSignedOdeTimJsonExpiration() {
    }
```
Comment (TonyEnglish) [inline jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java:627]: Fix line tab spacing

**Review by TonyEnglish — COMMENTED** [review]
Fix block on circleci

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java (hunk 2 above)
Comment (dan-du-car) [reply jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java:627]: Addressed

File: jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java (hunk 1 above)
Comment (dan-du-car) [reply jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java]: Addressed

**Comment by dan-du-car** [comment]
> Fix block on circleci

@TonyEnglish The circleCI is fixed

**Review by TonyEnglish — APPROVED** [review]
looks good, thx

**Pull request merged**